gubernator/templates/filtered_log.html

```diff
@@ -13,11 +13,14 @@
 <input type="hidden" name="logfile" value="{{log_file}}">
 <input type="hidden" name="pod" value="{{pod}}">
 <input type="hidden" name="junit" value="{{junit}}">
+<hr>
+<input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on
+<hr>
 <label><input type="checkbox" name="UID"{% if uid %} checked{% endif %}> Highlight Pod UID</label><br>
 <label><input type="checkbox" name="Namespace"{% if namespace %} checked{% endif %}> Highlight Namespace</label>
 </form>
 % if log
-<pre>
+{% if wrap %}<pre>{% else %}<pre class="nowrap">{% endif %}
```

Comment:
Not having lines wrap was a feature people asked for when I demo'd to my team yesterday, and we discussed reducing the length of the lines. The conclusion was that some didn't want the lines to be shortened by taking out the beginning, because they used the time stamps when debugging, and also that cutting the lines down may lead to code that needs to be maintained. But if there's a better solution that would be great.
/cc @timstclair